src/appmixer/utils/controls/OnError/OnError.js

```diff
@@ -1,59 +1,94 @@
 'use strict';
 
-async function getErrors(context, now, errors = [], from = 0, size = 100) {
-
-    const result = await context.callAppmixer({
-        endPoint: '/logs',
-        method: 'GET',
-        qs: {
-            from,
-            size,
-            sort: 'gridTimestamp:asc',
-            query: `@timestamp:[${context.state.since} TO ${now}] AND severity:error`,
-            flowId: context.flowId
-        }
-    });
+async function fetchErrorsPage(context, lock, limit, logId = null, lastGridTimestamp = null, size = 100, errors = []) {
```

Comment:
I think the `searchAfter` feature is only in Appmixer 6.0. But I cannot see a version dependency here (bundle.json).